pom.xml

```
@@ -41,14 +41,19 @@
 			<name>openHAB repository</name>
 			<url>https://dl.bintray.com/openhab/mvn</url>
 		</repository>
+		<repository>
+			<id>sympower-public</id>
+			<name>Sympower repository</name>
+			<url>https://extranet.sympower.net/nexus/repository/maven-public/</url>
```

Lets take the hostname from an environment variable as done in our other github projects, for example https://github.com/sympower/slf4j/blob/master/pom.xml#L380.

```suggestion
			<url>${env.MAVEN_REPO_URL}/maven-public/</url>
```

---

The repo definition you referenced as example is inside <distributionManagement> (which is invoked from CI). This place here is not. We do not want force every person building this project to define MAVEN_REPO_URL environment variable on their machine, do we?

---

Their impact is a bit different indeed. I would not mind that, though, as everyone from Sympower has a mirror defined in the settings.xml anyway, so an unresolved value would not affect us IMO. And I kind of liked the idea that our Nexus URL is not hardcoded anywhere, so the URL can be changed rather easily if needed: simply update the var once in CI and all projects still work. The same with devs.